service/src/test/java/bio/terra/cbas/dao/TestMethodDao.java

```diff
@@ -83,4 +83,11 @@ void retrievesAllMethods() {
     assertEquals(method2.methodId(), allMethods.get(0).methodId());
     assertEquals(method1.methodId(), allMethods.get(1).methodId());
   }
+
+  @Test
+  void unsetLastRunSetId() {
+    methodDao.unsetLastRunSetId(method1.methodId());
```

Comment:
This looks like it's already null in `method1`?

To prove me wrong, perhaps have an explicit "it's there!" check followed by the "unset" action and the "it's not there any more!" check

Reply:
+1 to Chris' comment

Reply:
thanks - careless of me